src/components/Comments/Comments.js

```diff
@@ -52,7 +52,8 @@ class Comments extends React.Component {
             <a className={classNames({ active: sort === 'OLDEST' })} data-type="OLDEST">Oldest</a>
           </div>
         </div>
-        <CommentForm />
+
+        {auth.isAuthenticated && <CommentForm username={auth.user.name} />}
```

Comment:
Tests fail because `auth` is `undefined`.
We should check if auth is not undefined like this `auth && auth.isAuthenticated && ...`